src/main/java/de/photon/anticheataddition/modules/sentinel/exploits/IllegalItemSentinel.java

```diff
@@ -35,7 +35,7 @@ private void handlePlayerEvent(Cancellable event, UUID uuid, ItemStack stack, Co
     {
         final var user = User.getUser(uuid);
         // User bypassed or stack nonexistent or not illegal.
-        if (User.isUserInvalid(user, this) || stack == null || !isIllegalStack(stack)) return;
+        if (User.isUserInvalid(user, this) || stack == null || !isIllegalStack(stack) || user == null) return;
```

Comment:
`isUserInvalid` already checks for null

Reply:
But code isn't working.
It can call NullPointerException.